force-app/main/default/objects/Opportunity/fields/Notification_Recipient_Information__c.field-meta.xml

```diff
@@ -5,8 +5,9 @@
     <externalId>false</externalId>
     <inlineHelpText>Enter any contact information such as address, phone, or email that the donor provided for the notification recipient.</inlineHelpText>
     <label>Notification Recipient Information</label>
-    <required>false</required>
+    <length>32768</length>
     <trackFeedHistory>false</trackFeedHistory>
     <trackTrending>false</trackTrending>
-    <type>TextArea</type>
+    <type>LongTextArea</type>
+    <visibleLines>10</visibleLines>
```

Comment:
This field type change should be fine when upgrading existing orgs, right?  I think this is the equivalent of modifying the length of a text field as mentioned here: https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/modifying_custom_fields.htm.  @grivero Is that your understanding also?

Reply:
@kenlewis-sfdo, this type of conversion is allowed and it's safe to update field type. 
This article also have good notes about field conversion: https://help.salesforce.com/s/articleView?id=sf.notes_on_changing_custom_field_types.htm&type=5